.cirrus/tasks.yml

```diff
@@ -15,8 +15,8 @@ env:
   GCLOUD_PRODUCT_NAME: official-sonarqube-data-center-edition
   GCLOUD_STAGING_REGISTRY: gcr.io/sonarqube-marketplace-provider
   GCLOUD_STAGING_PRODUCT_NAME: sonarqube-dce-staging
-  CURRENT_LTA_VERSION: 9.9.7
-  CURRENT_VERSION: 10.8.1
+  CURRENT_LTA_VERSION: 9.9.8
```

Comment:
should we put 2025.1 as CURRENT_LTA_VERSION as i think we had confirmation that no patch releases will be done only support.